packages/amplify_api/android/src/main/kotlin/com/amazonaws/amplify/amplify_api/FlutterApiRequest.kt

```diff
@@ -115,5 +115,25 @@ class FlutterApiRequest {
             }
         }
 
+        // ====== SHARED ======
```

Comment:
question: Would it make sense to omit the "SHARED" line here? I think this will only be used for GraphQL, correct? REST also uses an `apiName` but I think it is nested in another object and made available above by calling `getApiPath` (not sure why it has that name).